com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1175,6 +1175,8 @@ public void Shutdown(bool discardMessageQueue = false)
                 m_ShuttingDown = true;
                 m_StopProcessingMessages = discardMessageQueue;
             }
+
```

Comment:
It is annoying having to do this here. But when failing to host, neither `IsServer` nor `IsClient` is set. I also didn't want to move anything from `ShutdownInternal` to not risk introducing issues elsewhere.